src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -1,5 +1,37 @@
 package frc.robot.subsystems;
 
-public class Climber {
-    
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import frc.robot.Constants;
+
+public class Climber extends SubsystemBase {
+
+    CANSparkMax m_rightWinch;
+    CANSparkMax m_leftWinch;
+
+    public Climber() {
+        m_rightWinch = new CANSparkMax(Constants.CLIMBER_RIGHT_CAN_ID, MotorType.kBrushless);
+        m_leftWinch = new CANSparkMax(Constants.CLIMBER_RIGHT_CAN_ID, MotorType.kBrushless);
+    }
+
+    @Override
+    public void periodic() {
+    }
+
+    public void run(double rightSpeed, double leftSpeed) {
+        m_rightWinch.set(rightSpeed);
+        m_leftWinch.set(leftSpeed);
+    }
+
+    public void runLeft(double speed) {
```

Comment:
Why? Do you really see running 1 winch and not the other?

Reply:
yes if we are climbing on the side we may need to do one winch more than the other

Reply:
Yes, so you would set the speeds to be different. I don't see why you need to only set 1 motor and leave the other as is. If you are thinking "run left while right is stopped", then just use "run( leftSpeed, 0)".

I am being picky: extra code is generally a bad idea. It is a general rule of software engineering. Try to write only what is actually needed.

Reply:
lets talk during the meeting

Reply:
im worried that if we use two different buttons for the left and right winch we are going to run into issues with them overriding each other if they are using the same command

Reply:
im not the most experienced with this so it might be a non issue but still

Reply:
I agree that the command is going to be messy. It will need to be like the Drive command. It just needs to set both at the same time. One side might be 0 if you let go the trigger/stick.